remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingClient.java

```diff
@@ -210,8 +210,8 @@ public void initChannel(SocketChannel ch) throws Exception {
                             LOGGER.warn("Connections are insecure as SSLContext is null!");
                         }
                     }
-                    pipeline.addLast(
-                        defaultEventExecutorGroup,
```

Comment:
If null is passed in, the follow-up handlers would be skipped. This is a bug. 

See https://github.com/netty/netty/blob/d34212439068091bcec29a8fad4df82f0a82c638/transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java#L377